datafusion/functions-nested/src/make_array.rs

```diff
@@ -149,6 +153,39 @@ impl ScalarUDFImpl for MakeArray {
             )
         }
     }
+
+    fn documentation(&self) -> Option<&Documentation> {
+        Some(get_make_array_doc())
+    }
+}
+
+static DOCUMENTATION: OnceLock<Documentation> = OnceLock::new();
+
+fn get_make_array_doc() -> &'static Documentation {
+    DOCUMENTATION.get_or_init(|| {
+        Documentation::builder()
+            .with_doc_section(DOC_SECTION_ARRAY)
+            .with_description(
+                "Returns an Arrow array using the specified input expressions.",
```

Comment:
I think `Arrow` is superflous, though I suspect it came from the original text and is not incorrect. It might be clearer to omit

```suggestion
                "Returns an array using the specified input expressions.",
```